semantics.json

```diff
@@ -342,6 +342,14 @@
         "default": true
       },
       {
+        "name": "randomizeOnRetry",
+        "type": "boolean",
```

Comment:
But yes, I kind of assumed that if randomization of alternatives were off the new feature would also be off, but that might not be a valid assumption.

Maybe we should have a drop-down?
"Randomize alternatives (default)"
"Randomize alternatives and randomize again on retry"
"Do not randomize alternatives"